src/main/scala/nl/biopet/utils/tool/ToolDocumentation.scala

```diff
@@ -0,0 +1,163 @@
+package nl.biopet.utils.tool
+
+import java.io.{File, PrintWriter}
+
+import scala.io.Source
+
+trait ToolDocumentation {
```

Comment:
I think you also want to extend from Logging and add some logger statements inside the generation method